doc/lua_api.txt

```diff
@@ -8144,6 +8144,14 @@ Used by `minetest.register_node`.
         -- node "attaches". For the other drawtypes the image will be centered
         -- on the node.
 
+        visual_scale = {x = 1, y = 1, z = 1},
```

Comment:
Was the visual_scale property undocumented before?

The fact the property accept a number should be documented I think

Reply:
It's documented both for number and vector values as you can see.

Reply:
This is the deprecated table vector notation. Please change this to `vector.new(x, y, z)`.

Reply:
Done